homeassistant/components/bosch_shc/config_flow.py

```diff
@@ -59,7 +62,17 @@ async def async_step_user(self, user_input=None):
                 await self.async_set_unique_id(user_input[CONF_HOST])
                 self._abort_if_unique_id_configured()
 
-                return self.async_create_entry(title=info["title"], data=user_input)
+                mac_address = await self.async_get_mac(user_input[CONF_HOST])
```

Comment:
We prefer not to use getmac to get the mac address from the ip address since it doesn't work in all network environments.